src/components/common/ListPeopleAndRoles.vue

```diff
@@ -76,146 +76,148 @@
         </v-row>
 
         <!-- List Content -->
-        <v-row
-          v-for="(orgPerson, index1) in filteredPersonList"
```

Comment:
First, I moved the v-for into its own element (since can't have v-for and v-if in same element).

Second, I added v-if to conditionally show directors. This way, the v-for still iterates over the whole list and `index1` is retained, and the actions act on the correct list item.

Reply:
Example of short-form amalgamation (completing party only):

![image](https://github.com/bcgov/business-create-ui/assets/10002889/f1128c62-9712-45a9-9903-52be4e4a5a87)

Example of regular amalgamation (completing party is also a director):

![image](https://github.com/bcgov/business-create-ui/assets/10002889/ac48bed0-7ed6-4451-bf7e-2605ef5fcd6d)